CFE-473: Add information about profiling data lifetime (#52213)
openshift/openshift-docs: modules/node-observability-run-profiling-query.adoc

[id="running-profiling-query_{context}"]
= Running the profiling query

> To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data in the `emptyDir` volume. You can access the profiling data as long as a pod is in the `running` status.

Reviewer: ```suggestion
To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The data's life time is bound to the agent pod via the `emptyDir` volume, so you can access the profiling data as long as a pod is in the `running` status.
```

Author: Addressed

> To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The lifetime of data is bound to the agent pod through the `emptyDir` volume, so you can access the profiling data as long as the agent pod is in the `running` status.

Reviewer 2: Refer to ISG > Word Usage for [as long as](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as-long-as)
```suggestion
To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The lifetime of data is bound to the agent pod through the `emptyDir` volume, so you can access the profiling data while the agent pod is in the `running` status.
```

Author: Addressed
